commons/src/main/java/org/dhis2/commons/resources/D2ErrorUtils.kt

```diff
@@ -152,6 +152,9 @@ class D2ErrorUtils(
                 } else {
                     context.getString(R.string.error_no_internet_connection)
                 }
+
+            D2ErrorCode.DATABASE_IMPORT_FAILED -> TODO()
```

Comment:
could this lead to an exception? why don't we set a general error?

Reply:
Nope as the import/export database feature is not implemented. I think it would be best to implement it in the import/export ticket.